tests/inference/models_predictions_tests/test_owlv2.py

```diff
@@ -20,4 +20,62 @@ def test_owlv2():
     )
 
     response = OwlV2().infer_from_request(request)
-    assert abs(221.4 - response.predictions[0].x) < 0.1
+    # the exact value here is highly sensitive to the image interpolation mode used
+    # as well as the data type used in the model, ie bfloat16 vs float16 vs float32
+    # and of course the size of the model itself, ie base vs large
+    # we set a tolerance of 1.5 pixels from the expected value, which should cover most of the cases
+    assert abs(223 - response.predictions[0].x) < 1.5
+
+
+if __name__ == "__main__":
```

Comment:
I suggest moving somewhere else - I do maintain stack of useful scripts in `development` directory

Reply:
you gave me this comment and approved - should I interpret that as, move and rereview, or merge and keep this in mind for the future? I guess, what should my actionable be here?

thank you for the feedback! :)